src/wymeditor/editor/external-modules-browserify.js

```diff
@@ -5,5 +5,6 @@
 // task.
 WYMeditor.EXTERNAL_MODULES = {
     ObjectHistory: require("object-history"),
-    Combokeys: require("combokeys")
+    Combokeys: require("combokeys"),
+    Edited: require("edited")
 };
```

Comment:
For sake of debugging, this should be split up into one file per external module.